scripts/updateapilist.py

```diff
@@ -135,16 +243,17 @@ def client_row(client: CloudClient) -> str:
     url = f"https://github.com/{client.repo}"
     if client.repo == MONO_REPO:
         url += f"/tree/main/packages/{client.distribution_name}"
-
+    _show_api_issues = client.show_api_issues
+    _file_api_issue = client.file_api_issue
```

Comment:
(No action needed--just a comment)
You could also memoize the two properties on the RHS. The complication is that for this application, you need two pieces of information for the memoization (as you implemented above), and that's because the cached value could be `None`. In many applications, the memoized value is not `None`, so the value `None` can stand for "not yet cached".  So I don't think it's worth pursuing here, because it adds two fields to the class, and this is certainly clear enough.